cfgov/unprocessed/js/modules/beta-banner-state.js

```diff
@@ -5,38 +5,47 @@
 
 'use strict';
 
-// Import modules
-var $ = require( 'jquery' );
+// Required polyfills for <IE9.
+require( '../modules/polyfill/query-selector' );
+
+// Required modules.
+var Expandable = require( '../molecules/Expandable' );
 var _storage = require( './util/web-storage-proxy' );
 
 // Private variables.
-var _key = 'betaBannerIsCollapsed';
+var _expandable;
+
+// Constants.
+var COLLAPSED_STATE = 'betaBannerIsCollapsed';
+
 
 /**
  * Set up DOM references and event handlers.
  */
 function init() {
 
-  // DOM references.
-  var btnDOM = $( '#beta-banner_btn' );
-  var bannerDOM = $( '#beta-banner' );
+  // Init Expandable.
+  var betaBannerDom = document.querySelector( '#beta-banner' );
+  _expandable = new Expandable( betaBannerDom );
+  _expandable.init( _storage.getItem( COLLAPSED_STATE ) !== 'true' && 3 );
 
-  // Event handlers.
-  btnDOM.click( _betaBannerClicked );
-
-  // Initial state.
-  if ( _storage.getItem( _key ) !== 'true' ) {
-    bannerDOM.get( 0 ).expand();
-  }
+  initEvents();
 }
 
 /**
  * Run when the beta in-progress banner has been clicked.
  * @returns {boolean} always returns false.
  */
-function _betaBannerClicked() {
-  _toggleStorage();
-  return false;
+function initEvents() {
```

Comment:
Yeah, it should definitely be private but I was exposing it to make the code more testable for reasons mentioned here : http://philipwalton.com/articles/why-i-test-private-functions-in-javascript/. 

You shouldn't have to forgo encapsulation to do that though. I will make the change.